doc/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:20.04
+
+# Fix for tzdata for docker image build
+ENV TZ=UTC
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Install necessery for build packages
+RUN apt-get update && apt-get install -y \
+    apt-utils cpio python python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping \
+    python3-jinja2 libegl1-mesa pylint3 xterm \
+    vim locales devscripts debhelper \
+    gawk wget diffstat texinfo chrpath socat libsdl1.2-dev \
+    python-crypto checkpolicy python3-git python3-github \
+    bzr pigz m4 lftp openjdk-8-jdk git-core rsync \
+    gnupg flex bison gperf build-essential zip curl zlib1g-dev \
+    gcc-multilib g++-multilib libc6-dev-i386 lib32ncurses5-dev \
+    x11proto-core-dev libx11-dev lib32z1-dev ccache libgl1-mesa-dev \
+    libxml2-utils xsltproc unzip bc ninja-build \
+    python3-pyelftools python3-crypto libncurses5 libssl-dev -y
+
+RUN pip3 install pycryptodomex
+
+RUN apt-get install --reinstall -y ca-certificates
+
+# Clean up APT when done.
+RUN apt-get clean && rm -rf /tmp/* /var/tmp/*
```

Comment:
This makes very little sense, as each `RUN` command creates new "layer" (container image).  So you will clean contents of one layer, but all data will be preserved in the previous layers. This is why it is recommended to chain `RUN` commands: 

```
# Install necessery for build packages
RUN apt-get update && apt-get install -y \
    apt-utils cpio python python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping \
    python3-jinja2 libegl1-mesa pylint3 xterm \
    vim locales devscripts debhelper \
    gawk wget diffstat texinfo chrpath socat libsdl1.2-dev \
    python-crypto checkpolicy python3-git python3-github \
    bzr pigz m4 lftp openjdk-8-jdk git-core rsync \
    gnupg flex bison gperf build-essential zip curl zlib1g-dev \
    gcc-multilib g++-multilib libc6-dev-i386 lib32ncurses5-dev \
    x11proto-core-dev libx11-dev lib32z1-dev ccache libgl1-mesa-dev \
    libxml2-utils xsltproc unzip bc ninja-build \
    python3-pyelftools python3-crypto libncurses5 libssl-dev -y && \
    pip3 install pycryptodomex && \
    apt-get install --reinstall -y ca-certificates && \
    apt-get clean && rm -rf /tmp/* /var/tmp/*
```

Reply:
Done